### PR TITLE
Add isort to zeus

### PR DESCRIPTION
**Pull request by GarvinH**

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -11,4 +11,5 @@ pdfrw==0.4
 watchdog==0.9.0
 pytest===5.3.2
 pytest-django===3.9.0
-factory-boy===2.12.0
\ No newline at end of file
+factory-boy===2.12.0
```
Comment (AlexCLeduc) [inline dev-requirements.txt:14]: can we also bump pylint to 2.6.0 ? pylint 2.5.2  dependes on an older version of isort and makes pip install throw an error

File: zeus/versioning/core.py [hunk 2]
```diff
@@ -1,13 +1,9 @@
 import json
 
 from django.db import models
```
Comment (AlexCLeduc) [inline zeus/versioning/core.py:3]: we can apply the sort later, let's focus on the config in this PR

**Review by AlexCLeduc — CHANGES_REQUESTED** [review]
We should also add a deck in CI

```yaml
# .circleci/config.yml
  format_check:
    docker:
      - image: gcr.io/big-coil-207017/titan-ci:latest
        <<: *gcloud_auth
    steps:
      - attach_workspace:
          at: ./
      - run: env/bin/black --check ./
      - run: env/bin/isort # CHECK COMMAND? 

```

It looks like isort has a [command for this](https://github.com/PyCQA/isort#the---check-only-option), I'm not sure exactly which args we should pass 

We'll know the command works if this branch fails and the branch off this one that applies isort to all files passes

**Comment by AlexCLeduc** [comment]
Looks like isort is checking dependencies, 
Screen Shot 2021-03-11 at 11 00 51 AM


isort automatically ignores `venv/`. This is a good excuse to start using `venv/` instead, like every other python project (don't know why we ever settled on env). I've renamed the env upstream on master.

Locally, you'll have to reset your env, something like:

```bash
deactivate
rm -rf env/
python3 -m venv venv
```

Once you rebase, you can just replace the isort command to use `venv/bin/isort` instead.

**Comment by GarvinH** [comment]
I went ahead and updated pylint and pylint-django to their latest versions. Just thought I'd mention that in case

**Review by AlexCLeduc — APPROVED** [review]
Thanks! After merging, can you open another PR that applies isort to all the files?

**Pull request merged**